### ISSUE TITLE
Thumbs up/thumbs down fails a11y check

### ISSUE THREAD

**Issue by jhutchison-nava** [GSA/touchpoints#1870]
The thumbs up/thumbs down icons fail accessibility testing because they re-use their ID's

- https://github.com/GSA/touchpoints/blob/f2262ad210daa0201b4a62b6674cd259ed23cb97/app/views/components/forms/question_types/_big_thumbs_up_down_buttons.html.erb#L39-L42
- https://github.com/GSA/touchpoints/blob/f2262ad210daa0201b4a62b6674cd259ed23cb97/app/views/components/forms/question_types/_big_thumbs_up_down_buttons.html.erb#L61-L64

Here's an excerpt from our pa11y (axe) results:

```
[
 {
        "code": "WCAG2AA.Principle4.Guideline4_1.4_1_1.F77",
        "type": "error",
        "typeCode": 1,
        "message": "Duplicate id attribute value \"thumbs-up-icon\" found on the web page.",
        "context": "<desc id=\"thumbs-up-icon\">\n                Thumbs-up icon...</desc>",
        "selector": "#thumbs-up-icon",
        "runner": "htmlcs",
        "runnerExtras": {}
      },
      {
        "code": "WCAG2AA.Principle4.Guideline4_1.4_1_1.F77",
        "type": "error",
        "typeCode": 1,
        "message": "Duplicate id attribute value \"thumbs-down-icon\" found on the web page.",
        "context": "<desc id=\"thumbs-down-icon\">\n                Thumbs-down ic...</desc>",
        "selector": "#thumbs-down-icon",
        "runner": "htmlcs",
        "runnerExtras": {}
      },
      {
        "code": "duplicate-id-aria",
        "type": "error",
        "typeCode": 1,
        "message": "IDs used in ARIA and labels must be unique (https://dequeuniversity.com/rules/axe/4.2/duplicate-id-aria?application=axeAPI)",
        "context": "<title id=\"thumbs-up-icon\">Thumbs-up</title>",
        "selector": "#thumbs-up-icon",
        "runner": "axe",
        "runnerExtras": {
          "description": "Ensures every id attribute value used in ARIA and in labels is unique",
          "impact": "critical",
          "help": "IDs used in ARIA and labels must be unique",
          "helpUrl": "https://dequeuniversity.com/rules/axe/4.2/duplicate-id-aria?application=axeAPI"
        }
      },
      {
        "code": "duplicate-id-aria",
        "type": "error",
        "typeCode": 1,
        "message": "IDs used in ARIA and labels must be unique (https://dequeuniversity.com/rules/axe/4.2/duplicate-id-aria?application=axeAPI)",
        "context": "<title id=\"thumbs-down-icon\">Thumbs-down</title>",
        "selector": "#thumbs-down-icon",
        "runner": "axe",
        "runnerExtras": {
          "description": "Ensures every id attribute value used in ARIA and in labels is unique",
          "impact": "critical",
          "help": "IDs used in ARIA and labels must be unique",
          "helpUrl": "https://dequeuniversity.com/rules/axe/4.2/duplicate-id-aria?application=axeAPI"
        }
      }
    ],

**Comment by jhutchison-nava** [comment]
Hi @rileyseaburg . Should we be seeing this update on our forms by now? Is there something we should be doing on our end to see this change?
